src/App.jsx

```diff
@@ -1,8 +1,17 @@
-import React from "react";
-import "./App.css";
+import React, { useEffect, useState } from "react";
+import "src/assets/styles/styles.scss";
+import SplashScreen from "src/components/splashScreen/SplashScreen";
+import Body from "src/components/body/Body";
 
 function App() {
-  return <></>;
+  const [isLoading, setIsLoading] = useState(true);
+  useEffect(() => {
+    setTimeout(() => {
+      setIsLoading(false);
+    }, 1000);
```

Comment:
Why are we setting 1 sec of loading here?

Reply:
@fza-wtag removed the timeout